CHANGELOG.md

```diff
@@ -1,122 +1,124 @@
-- The `bPackRPCs` property has been removed, and the flag `--OverrideRPCPacking` has been removed.
-- Added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and ActorComponents. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
-- You can now generate valid schema for classes that start with a leading digit. The generated schema class will be prefixed with `ZZ` internally.
-- Handover properties will be automatically replicated when required for load balancing. `bEnableHandover` is off by default.
-- Added `OnSpatialPlayerSpawnFailed` delegate to `SpatialGameInstance`. This is helpful if you have established a successful connection but the server worker crashed.
+- We've added `bEnableClientQueriesOnServer` (default false) which makes the same queries on the server as it makes on clients, if the unreal load balancer is enabled. Enable this to avoid a situation in which clients receive updates about entities that the server doesn't receive updates about (if the server's interest query is configured incorrectly).
+- We've added log warning when `AddPendingRPC` fails due to `ControllerChannelNotListening`.
+- When offloading is enabled, Actors have local authority (`ROLE_Authority`) on servers for longer periods of time, to allow more native Unreal functionality to work without problems.
+- When offloading is enabled, if you try to spawn Actors on a server that will not be the Actor Group owner for them, we now log an error and delete the Actor.
+- The GDK now uses SpatialOS Runtime version 14.5.1 by default.
+- We've renamed the configuration setting `bPreventAutoConnectWithLocator` to `bPreventClientCloudDeploymentAutoConnect` and moved it to GDK Settings. If using this feature please update enable the setting in GDK Settings.
+- We've made `USpatialMetrics::WorkerMetricsRecieved` static.
+- You can now connect to a local deployment by selecting "Connect to a local deployment" and specifying the local IP address of your computer in the Launch drop-down menu.
+- We've enabled RPC ring buffers by default, and we'll remove the legacy RPC mode in a future release.
+- We've removed the `bPackRPCs` property and the the `--OverrideRPCPacking` flag.
+- We've added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and Actor Components. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
+- You can now generate valid schema for classes that start with a leading digit. The generated schema class are prefixed with `ZZ` internally.
+- The GDK now automatically replicates handover properties when this is required for load balancing. `bEnableHandover` is off by default.
+- We've added `OnSpatialPlayerSpawnFailed` delegate to `SpatialGameInstance`. This is helpful if you established a successful connection but the server-worker instance crashed.
```

Comment:
"delegate” again (I don’t know what this means, but might just be my lack of knowledge)